dojo/models.py

```diff
@@ -1548,12 +1547,7 @@ def sla(self):
         return sla_calculation
 
     def jira(self):
-        try:
-            jissue = JIRA_Issue.objects.get(finding=self)
-        except:
-            jissue = None
-            pass
-        return jissue
+        return self.jira_issue
```

Comment:
I think this will fail if there is no JIRA issue?

Reply:
Ah yes, good point. This needs to catch `DoesNotExist`.